src/psyclone/configuration.py

```diff
@@ -1314,11 +1326,13 @@ def get_loop_type_data(self):
         return self._loop_type_data
 
     def get_valid_loop_types(self):
-        '''
-        :returns: a list of valid loop types.
+        '''The list is sorted to have reproducible results for testing.
```

Comment:
Please put a new line between the opening ''' and the sentence and then please follow the sentence with a blank line.

Reply:
Done.